internal/app/connectconformance/testsuites/client_message_size.yaml

```diff
@@ -0,0 +1,251 @@
+name: Client Message Size
+mode: TEST_MODE_CLIENT
+reliesOnMessageReceiveLimit: true
+# TODO - The size limit is currently only calculated against the proto size
+relevantCodecs:
+  - CODEC_PROTO
```

Comment:
@jhump the request expansion logic seems to just use the proto codec for size calculations. I'm not sure if we should also test at least the JSON codec, but my hunch is that testing with proto only is probably fine?

Reply:
Ah, yes. Sorry I forgot to mention it to you. It may be worth adding a check when we load the test case library to complain if it sees a request that uses the "expand request" stuff but with a codec _other than_ proto.